<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>polap</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
</head>
<body>
<pre><code>POLAP - Plant organelle DNA long-read assembly pipeline.
version 0.2.4

Usage: polap &lt;menu&gt; [&lt;menu2&gt; [&lt;menu3&gt;]] [-l|--long-reads &lt;arg&gt;] [-o|--outdir &lt;arg&gt;] [-a|--short-read1 &lt;arg&gt;] [-b|--short-read2 &lt;arg&gt;] [--sra &lt;arg&gt;] [-p|--unpolished-fasta &lt;arg&gt;] [-f|--final-assembly &lt;arg&gt;] [-m|--min-read-length &lt;arg&gt;] [-t|--threads &lt;arg&gt;] [-c|--coverage &lt;arg&gt;] [-r|--pair-min &lt;arg&gt;] [-x|--bridge-min &lt;arg&gt;] [-w|--single-min &lt;arg&gt;] [-i|--inum &lt;arg&gt;] [-j|--jnum &lt;arg&gt;] [-g|--genomesize &lt;arg&gt;] [--bioproject &lt;arg&gt;] [--species &lt;arg&gt;] [--accession &lt;arg&gt;] [--query &lt;arg&gt;] [--subject &lt;arg&gt;] [-M|--minimum &lt;arg&gt;] [--(no-)reduction-reads] [--(no-)contigger] [--(no-)all-annotate] [--(no-)use-edges] [--(no-)coverage-check] [--(no-)resume] [-u|--(no-)circularize] [--(no-)test] [-v|--version] [-h|--help]
       polap &lt;menu&gt; help

menu: list, make-menus, or clean-menus
  assemble1, annotate, assemble2, flye-polishing,
  reset, total-length-long, find-genome-size, reduce-data,
  flye1, blast-genome, count-gene, select-reads, flye2,
  flye-polishing, check-coverage,
  prepare-polishing, polish,
  assemble,



Menu: assemble1

  polap assemble1 [-o|--outdir &lt;arg&gt;] [-l|--long-reads &lt;arg&gt;] [-a|--short-read1 &lt;arg&gt;] [-b|--short-read2 &lt;arg&gt;] [-m|--min-read-length &lt;arg&gt;] [-t|--threads &lt;arg&gt;] [-c|--coverage &lt;arg&gt;]

  or

  polap reset [-o|--outdir &lt;arg&gt;]

  polap total-length-long [-l|--long-reads &lt;arg&gt;]

  polap find-genome-size  [-a|--short-read1 &lt;arg&gt;] [-b|--short-read2 &lt;arg&gt;]

  polap reduce-data [-l|--long-reads &lt;arg&gt;] [-m|--min-read-length &lt;arg&gt;]

  polap flye1 [-t|--threads &lt;arg&gt;] [-c|--coverage &lt;arg&gt;] [-g|--genomesize &lt;arg&gt;]



Menu: annotate

  polap annotate [-i|--inum &lt;arg&gt;]

  or

  polap blast-genome [-i|--inum &lt;arg&gt;]

  polap count-genes [-i|--inum &lt;arg&gt;]



Menu: assemble2

  polap assemble2 [-o|--outdir &lt;arg&gt;] [-l|--long-reads &lt;arg&gt;] [-a|--short-read1 &lt;arg&gt;] [-b|--short-read2 &lt;arg&gt;] [-m|--min-read-length &lt;arg&gt;] [-t|--threads &lt;arg&gt;] [-c|--coverage &lt;arg&gt;]

  or

  polap select-reads [-i|--inum &lt;arg&gt;] [-j|--jnum &lt;arg&gt;] [-r|--pair-min &lt;arg&gt;] [-x|--bridge-min &lt;arg&gt;] [-w|--single-min &lt;arg&gt;]

  polap flye2 [-j|--jnum &lt;arg&gt;] [-t|--threads &lt;arg&gt;] [-c|--coverage &lt;arg&gt;]



Menu: polishing

  polap flye-polishing  [-j|--jnum &lt;arg&gt;] [-t|--threads &lt;arg&gt;] [-c|--coverage &lt;arg&gt;]

  polap prepare-polishing  [-a|--short-read1 &lt;arg&gt;] [-b|--short-read2 &lt;arg&gt;]

  polap polish [-p|--unpolished-fasta &lt;arg&gt;] [-f|--final-assembly &lt;arg&gt;]



Options:

  -o, --outdir: output folder name (default: o)

  -l, --long-reads: long-reads data file in fastq format (default: &#39;l.fq&#39;)

  -a, --short-read1: short-read fastq file 1 (default: &#39;s1.fq&#39;)

  -b, --short-read2: short-read fastq file 2 (default: &#39;s2.fq&#39;)

  -p, --unpolished-fasta: polishing sequence in fasta format (default: &#39;mt.0.fasta&#39;)

  -f, --final-assembly: final assembly in fasta format (default: &#39;mt.1.fa&#39;)

  -m, --min-read-length: minimum length of long reads (default: &#39;3000&#39;)

  -t, --threads: number of CPUs (default: &#39;56&#39;)

  -c, --coverage: coverage for the 2nd assembly (default: &#39;30&#39;)

  -r, --pair-min: minimum mapped bases or PAF 11th column (default: &#39;3000&#39;)

  -x, --bridge-min: minimum bridging read length or PAF 7th column (default: &#39;3000&#39;)

  -w, --single-min: minimum mapped bases or PAF 11th column (default: &#39;3000&#39;)

  -i, --inum: previous output number of organelle-genome assembly (default: &#39;0&#39;)

  -j, --jnum: current output number of organelle-genome assembly (default: &#39;1&#39;)

  -g, --genomesize: expected genome size (no default)

  -u, --circularize, --no-circularize: circularize a contig (off by default)

  -v, --version: Prints version

  -h, --help: Prints help



Places your long-read and short-read files at a folder.

long-read file [l.fq]

short-read files [s1.fq] and [s2.fq]

Execute: polap reset</code></pre>
</body>
</html>
